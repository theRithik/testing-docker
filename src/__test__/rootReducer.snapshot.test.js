import React from 'react';
import rootReducer from '../reducer/mainReducer';
import {create} from 'react-test-renderer'

describe('Snapshot Test',()=>{
    test('testing rootReducer',()=>{
        let tree= create(<rootReducer/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})