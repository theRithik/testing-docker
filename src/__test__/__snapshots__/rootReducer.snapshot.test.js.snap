// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test testing rootReducer 1`] = `<rootReducer />`;
